src/main.rs

```diff
@@ -83,6 +83,7 @@ fn main() {
 
         // OpenID Connect relying party endpoints
         get "/callback" => broker::handlers::oauth2::Callback::new(&app),
+        post "/callback" => broker::handlers::oauth2::Callback::new(&app),
```

Comment:
I tried to find it in the spec, but I did not find a clear statement that the implicit flow has to use a POST here. I missed it, or that is just how Google does it?